### ISSUE TITLE
False negative with moving numbers which implements `Copy`

### ISSUE THREAD

**Issue by p0lunin** [intellij-rust/intellij-rust#6487]
## Environment

* **IntelliJ Rust plugin version:** 0.3.131.3366-201
* **Rust toolchain version:** 1.47.0 (18bf6b4f0 2020-10-07) x86_64-pc-windows-msvc
* **IDE name and version:** PyCharm 2020.1.2 Community Edition (PC-201.7846.77)
* **Operating system:** Windows 10 10.0

## Problem description
When i move twice numbers which implements `Copy` trait i got an error:

![image](https://user-images.githubusercontent.com/48922415/101241105-0183b780-36fc-11eb-99a4-a94a3885b51f.png)

## Steps to reproduce
```rust
fn foo(x: i32) {
    let a = x;
    let b = x;
}
```

**Comment by ortem** [comment]
Please try executing `Re-Expand All Rust Macros` action, and then `Invalidate caches` action if the first one doesn't help. Both actions can be found using ctrl+shift+A menu.
